pkg/skaffold/deploy/kubectl.go

```diff
@@ -230,33 +174,6 @@ func (k *KubectlDeployer) readRemoteManifest(name string) ([]byte, error) {
 	return manifest.Bytes(), nil
 }
 
-func generateManifest(b build.Build) ([]byte, error) {
-	logrus.Info("No manifests specified. Generating a deployment.")
-
-	dockerfilePath := filepath.Join(b.Artifact.Workspace, b.Artifact.DockerArtifact.DockerfilePath)
-	r, err := os.Open(dockerfilePath)
-	if err != nil {
-		return nil, errors.Wrap(err, "reading dockerfile")
-	}
-
-	ports, err := docker.PortsFromDockerfile(r)
```

Comment:
You should also be able to delete this function from the docker package: `PortsFromDockerfile` and tests.

Reply:
Thanks @dlorenc I removed it too now